File: web3rpc/rpc-specs/code-samples/python/debug/blockchainInspection/getModifiedAccountsByHash.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

startBlockHash = "0xcc1ac1e244f9f83b812b5d77ada1c399f02ea7b61e72ff31789d9ef6dad45442"
endBlockHash = "0x437e92b2d30a0a828dfdd23b837a8ddf8c8b79c222e191d16c47afbf5a6aaed7"

w3 = Web3(Web3.HTTPProvider(host))
debug_response = w3.debug.get_modified_accounts_by_hash(startBlockHash, endBlockHash)

print(debug_response)

File: web3rpc/rpc-specs/code-samples/javascript/debug/blockchainInspection/getModifiedAccountsByHash.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
  const startBlockHash =
    "0x165c29e453dff6e1d9838d9e975a438b6f11a2c0a281b0d5b97c8d3110a79ac5";
  const endBlockHash =
    "0x2a8acdc3e9bb735918dc6a0141b9939976f446fde0b39336d74278da93b8d41d";

  provider.debug
    .getModifiedAccountsByHash(
      startBlockHash,
      { endBlockHash },
      (err, data, response) => {},
    )
    .then((data) => {
      console.log(data);
    });
})();
